lib/stages/bufferRecv.cpp

```diff
@@ -214,6 +214,15 @@ void bufferRecv::main_thread() {
     listener = socket(AF_INET, SOCK_STREAM, 0);
     evutil_make_socket_nonblocking(listener);
 
+    // This makes is possible to reuse the same socket even if it goes into TIME_WAIT
+    // Used by gossec
+    int reuse = 1;
+    if (setsockopt(listener, SOL_SOCKET, SO_REUSEADDR, &reuse, sizeof(int)) < 0) {
+        ERROR("Could not set SO_REUSEADDR on socket");
+        close(listener);
+        return;
+    }
```

Comment:
@andrerenard could you comment on this change? It is needed to make it possible for `gossec` to restart an instance of `kotekan` listening on the same port without running into `TIME_WAIT`

Reply:
To be honest I don't have a great understanding of what implications option has...   I would suggest for now we only use it if `drop_frames` is false.  I don't want to risk creating an unexpected edge case in the normal use of `bufferRecv`